scripts/setupVitest.ts

```diff
@@ -1,4 +1,4 @@
-import { vi } from 'vitest'
+import { vi, expect, beforeEach, afterEach } from 'vitest'
```

Comment:
```suggestion
/// <reference types="vitest/globals" />
```

Or we can just add types reference since Vitest can work without importing
https://github.com/vuejs/core/blob/98f1934811d8c8774cd01d18fa36ea3ec68a0a54/vitest.config.ts#L24

Reply:
Thanks for the suggestion，I refactored it. Can it be merged